backend/src/app/rpc/commands/files/create.clj

```diff
@@ -64,13 +65,16 @@
 
     (files/decode-row file)))
 
+(s/def ::nilable-page-id (s/nilable ::us/uuid))
+
 (s/def ::create-file
   (s/keys :req-un [::files/profile-id
                    ::files/name
                    ::files/project-id]
           :opt-un [::files/id
                    ::files/is-shared
-                   ::files/features]))
+                   ::files/features
+                   ::nilable-page-id]))
```

Comment:
This will not going to validate never the `:page-id` attr.
Also you are touching `create-file` public API, when you really need to touch the create-temp-file.

Reply:
I think it's useful to be able to create a file without automatically adding a blank page, both in create-file and in create-tem-file (although we only use it for the moment in the second one).

I have more doubts in using the nil value of page-id or adding another paramenter like `:avoid-auto-page` or similar. I think it validates well: you can omit the `:page-id` attribute, or send it with null value.

Reply:
changed